test/dynamo/test_graph_input_matcher.py

```diff
@@ -0,0 +1,60 @@
+import unittest
```

Comment:
you need to add this test to the `test/run_test.sh`. Let's do that in a follow up pr. I will merge this one to unblock hexllm folks

Reply:
Ok, sorry missed this comment yesterday, thanks for adding it in a new PR @JackCaoG !